bee-tangle/src/lib.rs

```diff
@@ -7,15 +7,16 @@ use tokio::sync::RwLock;
 
 use std::{collections::HashMap, sync::Arc};
 
-struct MessageData {
-    message: Arc<Message>,
-    metadata: MessageMetadata,
+/// Stores a [`Message`] with its [`MessageMetadata`].
+pub struct MessageData {
+    pub message: Message,
```

Comment:
Please put getters instead of public fields

Reply:
fixed